packages/editor/package.json

```diff
@@ -34,6 +34,7 @@
 		"@babel/runtime": "^7.16.0",
 		"@wordpress/a11y": "file:../a11y",
 		"@wordpress/api-fetch": "file:../api-fetch",
+		"@wordpress/autop": "^4.7.0",
```

Comment:
Why do we need this? This is a sort of legacy package that should be avoided. I don't understand why it's needed.

Reply:
Based on [feedback](https://github.com/WordPress/gutenberg/pull/60622#discussion_r1736569516), we have added this package.